### PR TITLE
Changes to the main README file

### PR DESCRIPTION
**Pull request by shansrin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,25 +8,35 @@ http://oss.oracle.com/licenses/upl.
...
 
-We have a **public Slack channel** where you can get in touch with us to
-ask questions about using the operator or give us feedback or
-suggestions about what features and improvements you would like to see.
-We would love to hear from you. To join our channel, please [visit this site to get an invitation](https://join.slack.com/t/oraclecoherence/shared_invite/enQtNjA3MTU3MTk0MTE3LWZhMTdhM2E0ZDY2Y2FmZDhiOThlYzJjYTc5NzdkYWVlMzUzODZiNTI4ZWU3ZTlmNDQ4MmE1OTRhOWI1MmIxZjQ).  The
-invitation email will include details of how to access our Slack
-workspace.  After you are logged in, please come to `#operator` and say,
-"hello!"
+* Run Coherence within the industry standard Kubernetes container orchestration framework, using Docker containers for the members of a Coherence cluster.
+* Use popular industry standard tools such as Grafana, ELK (or more specifically the Elasticsearch, Fluentd and Kibana (EFK) stack, and Prometheus to monitor the performance, logs, and health of your clusters.
```
Comment (harveyraja) [inline README.md]: Missing closing bracket after '(EFK) stack'.

File: README.md [hunk 2]
```diff
@@ -8,25 +8,35 @@ http://oss.oracle.com/licenses/upl.
...
+* Flexibly override and customize cluster configuration.
+* Scale the Coherence deployment.
```
Comment (harveyraja) [inline README.md]: Using kubernetes verbs and/or updates, scaler the Coherence deployment.

Comment (shansrin) [reply README.md]: Can you explain more on the issue?

File: README.md (hunk 1 above)
Comment (shansrin) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (fryp) [reply README.md]: I think Harvey means change the last bullet to say something like: "Scale the Coherence deployment using Kubernetes verbs and/or updates."  Or "Using Kubernetes verbs and/or updates, scale the Coherence deployment."

File: README.md [hunk 3]
```diff
@@ -8,25 +8,35 @@ http://oss.oracle.com/licenses/upl.
 
 -----
 
-# Public Slack Channel Available
+# Oracle Coherence Operator
```
Comment (rstafford) [inline README.md]: Please drop the word Oracle.  After multiple discussions on naming, we have consensus on "Coherence Operator" and we want the nomenclature to be consistent everywhere.

File: README.md (hunk 2 above)
Comment (shansrin) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (shansrin) [reply README.md]: fixed

**Pull request merged**